### DISCUSSION TITLE
Welcome to Oqtane Discussions!

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#677] [General]
Our community has been included in the Beta for Github Discussions! We will be evaluating this new feature to determine if it meets our needs better than the current Gitter room for discussions. Please utilize the many features to:

  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and always be open-minded and respectful. 

Remember that this is a community we build together...

**Comment by ADefWebserver** [comment]
I vote for this over Gitter 👍 

- [X] I can selectively subscribe/unsubscribe
- [X] I can actually follow the discussions!

**Comment by thabaum** [comment]
awesome!

**Comment by markdav-is** [comment]
I'm really digging this so far.  I'll have to check on getting some other projects in the Beta
